.gitmodules

```diff
@@ -31,3 +31,12 @@
 [submodule "external/alfons"]
 	path = external/alfons
 	url = https://github.com/hjanetzek/alfons
+[submodule "external/curl"]
+	path = external/curl
+	url = https://github.com/lygstate/curl.git
+[submodule "external/zlib"]
+	path = external/zlib
+	url = https://github.com/madler/zlib.git
+[submodule "external/angle"]
+	path = external/angle
+	url = https://github.com/lygstate/angle.git
```

Comment:
The build process doesn't appear to use this repository, only the build product (libGLESv2). If that is the case then 1. we should say in the README that users should build ANGLE through a separate build process and 2. we don't need this to be a submodule in tangram-es.